src/main/java/org/mamute/auth/LDAPApi.java

```diff
@@ -87,6 +89,7 @@ public void init() {
 			groupAttr = env.get(LDAP_GROUP, "");
 			moderatorGroup = env.get(LDAP_MODERATOR_GROUP, "");
 			lookupAttrs = env.get(LDAP_LOOKUP, "").split(",");
+			useSsl = Boolean.parseBoolean(env.get(LDAP_USE_SSL, "false"));
```

Comment:
I am out of the office until 13.04.2015.

In dringenden F�llen, kontaktieren Sie bitte
Erwin<punkt>Tratar<at>gebit<punkt>de oder
Tom<punkt>Krauss<at>gebit<punkt>de.


Note: This is an automated response to your message  "Re: [mamute] Adding
LDAP SSL option (#152)" sent on 08.04.2015 22:24:34.

This is the only notification you will receive while this person is away.